airbyte-integrations/connectors/source-slack/source_slack/components/channel_members_extractor.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+
+from dataclasses import dataclass
+from typing import List
+
+import requests
+from airbyte_cdk.sources.declarative.extractors import DpathExtractor
+from airbyte_cdk.sources.declarative.types import Record
+
+
+@dataclass
+class ChannelMembersExtractor(DpathExtractor):
```

Comment:
We should properly handle errors from slack. like it was before:

Reply:
Could you please give me an example that is not covered in this handling errors implementation? Now it works the same  as in https://github.com/airbytehq/airbyte/pull/35477#discussion_r1535888744, but instead of AirbyteTracedException it raises airbyte_cdk.sources.declarative.exceptions.ReadException.